Tests/store_packs.py

```diff
@@ -146,7 +146,8 @@ def _parse_pack_metadata(user_metadata, pack_content_items, pack_id):
         is_deprecated = user_metadata.get('deprecated', False)
         pack_metadata['deprecated'] = bool(strtobool(is_beta)) if isinstance(is_deprecated, str) else is_deprecated
         pack_metadata['certification'] = user_metadata.get('certification', '')
-        pack_metadata['price'] = int(user_metadata.get('price', 0))
+        price = user_metadata.get('price', 0)
+        pack_metadata['price'] = int(price) if isinstance(price, int) else 0
```

Comment:
price will not be an `int`, maybe try/except?

Reply:
I will improve it anywhere in the future, so no worries